client/src/store.js

```diff
@@ -222,5 +223,32 @@ export const useStore = defineStore("store", {
           });
       });
     },
+    userUnAuth(router) {
+      return new Promise((resolve, reject) => {
+        fetch(`http://localhost:8080/auth/logout`, {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            accessToken: this.accessToken,
+          }),
+        })
+          .then((response) => {
+            console.log(response);
+            if (response.ok) {
+              router.push({
+                name: "auth",
+              });
+              this.isAuthenticated = false;
+            } else {
+              reject(new Error(response.status + " error"));
+            }
+          })

```

Comment:
Dodal sem novo komponento NavBarAuth, kjer se uporabniku prikaže Logout ali Signout, odvisno ali je že v DB, ali pač ne.
Za logout sem uporabil `post `metodo, kjer se jwt token, ki ga dobi določen uporabnik, filitrira z filter metodo z vsemi tokensi znotraj `accestokens `arraya, in če je token uporanika enak z tokenom v arrayu, potem vrne accestokens array brez tega tokensa, kar pomeni, da je tokens izbrisan in je uporabnik logout.

Reply:
zlo specifično za tvojo implementacijo tukej ampak je uredu.. načeloma bi logout bil samo `get` kjer se zbriše token / cookie / session..